### ISSUE TITLE
Update peerDependencies for react 16

### ISSUE THREAD

**Issue by nassimbenkirane** [patw0929/react-intl-tel-input#189]
Hello !

Thanks for this library 👍 
When using react-intl-tel-input with react16, yarn gives me those warnings :

```
warning " > react-intl-tel-input@4.3.1" has incorrect peer dependency "prop-types@~15.5.8".
warning " > react-intl-tel-input@4.3.1" has incorrect peer dependency "react@^15.4.2".
warning " > react-intl-tel-input@4.3.1" has incorrect peer dependency "react-dom@^15.4.2".
```

Would it be possible to update the dependencies ? Are they known issues with React 16 that prevents this ?

Thanks a lot :)

**Comment by nassimbenkirane** [comment]
PS, I can do a PR if you want

**Comment by patw0929** [comment]
Ah, sorry for the late reply, it will be appreciated if you could help. 😄 
I think it might be a breaking change to upgrade to react 16.

Thank you! 👍

**Comment by patw0929** [comment]
Already merge #196 and released as v5.0.0 to support react 16.
If it still has problem, please reopen this issue, thanks!

**Comment by nassimbenkirane** [comment]
Ah, sorry I didn't reply earlier !

Thanks a lot for the update 👍
